src/main/java/com/hedera/exchange/Rate.java

```diff
@@ -249,4 +249,12 @@ public Rate clipRate(final Rate newRate, long bound) {
     public String toJson() throws JsonProcessingException {
         return OBJECT_MAPPER.writeValueAsString(this);
     }
+
+    @JsonIgnore
+    /**
+     * Get the hbar rate in USD
+     */
+    public double getRateinUSD() {
```

Comment:
This method doesn't appear to have unit tests

Reply:
Done